### ISSUE TITLE
Embed Javascript Widget

### ISSUE THREAD

**Issue by reustle** [reustle/covid19japan#35]
We currently have https://covid19japan.com/embed which allows news sites to embed the map directly into their page. Since iframes are difficult, the content is either cut off or too wide. There are [many resources](https://stackoverflow.com/questions/17838607/making-an-iframe-responsive) on how to create responsive iframes. I feel the widget should work as follows

Provide some kind of javascript snippet such as 
``` html
<script src="https://covid19japan.com/embed.js"></script>
```

Which contains roughly the following code
``` javascript
document.write('<div id="covid19japan-embed"><iframe src="https://covid19japan.com/embed"/></div>')
document.getElementById('covid19japan-embed').style = // TODO
```

Basically we create a wrapper element, load the iframe within it, and set the wrapper style to handle the responsiveness properly.

**Comment by reustle** [comment]
From @python273 regarding sizing the iframe - https://stackoverflow.com/questions/9162933/make-iframe-height-dynamic-based-on-content-inside-jquery-javascript/23020025#23020025
